winml/lib/Api.Image/CpuTensorizer.h

```diff
@@ -33,6 +34,19 @@ class CpuTensorizer {
     uint32_t xElements = inputBounds.Width - inputBounds.X;
     uint32_t yElements = inputBounds.Height - inputBounds.Y;
 
+    uint32_t channelNum = formatFrom == kImageTensorChannelTypeGRAY8 ? 1 : 3;
+    uint32_t totalElements = channelNum * inputBounds.Width * inputBounds.Height;
+
+    float scale = 1.0; 
+    int32_t shift = 0;
+    if (pixelRange == ImageNominalPixelRange::kNormalized_0_1) {
+      scale = float(1.0) / 255;
+      shift = 0;
+    } else if (pixelRange == ImageNominalPixelRange::kNormalized_1_1) {
+      scale = float(1.0) / 255 * 2;
```

Comment:
>255 * 2 [](start = 27, length = 7)

scale = (2.f / 255.f); // add a comment here to explain why this math works and its crude derivation as related to the 11 normalization scheme